components/sysutils/rdiff-backup/Makefile

```diff
@@ -19,36 +19,59 @@
 # CDDL HEADER END
 
 #
+# Copyright 2021 Gary Mills
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		rdiff-backup
-COMPONENT_VERSION=	1.3.3
-COMPONENT_REVISION=	2
+COMPONENT_VERSION=	2.0.5
+# COMPONENT_REVISION=	0
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE=	v$(COMPONENT_VERSION).tar.gz
 COMPONENT_ARCHIVE_HASH=	\
-    sha256:ee030ce638df0eb1047cf72578e0de15d9a3ee9ab24da2dc0023e2978be30c06
-COMPONENT_ARCHIVE_URL=	http://download.savannah.gnu.org/releases/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
-COMPONENT_PROJECT_URL=  http://www.nongnu.org/rdiff-backup/
+    sha256:fd3af3d39fe91ef2b281e6a906445112c5b718640e490c8b6f83a8960318352b
+COMPONENT_ARCHIVE_URL=	https://github.com/$(COMPONENT_NAME)/$(COMPONENT_NAME)/archive/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=  https://rdiff-backup.net/
 COMPONENT_BUGDB=	utility/rdiff-backup
 
-PYTHON_VERSIONS=	2.7
+# This version of rdiff-backup supports python versions: 3.5 3.6 3.7 3.8
+PYTHON_VERSIONS=	3.7
+
+# Copy static files to destination directories
+COMPONENT_POST_INSTALL_ACTION = ( \
+cd $(PROTO_DIR); \
+$(MKDIR) -m 0755 usr/share/doc/rdiff-backup-$(COMPONENT_VERSION) \
+  usr/share/bash-completion/completions usr/share/man/man1; \
+for V in $(PYTHON_VERSIONS); \
+do \
+  for N in CHANGELOG.md COPYING FAQ.md README.md; \
+  do \
+    $(CP) -p usr/lib/python$$V/vendor-packages/share/doc/rdiff-backup/$$N \
+      usr/share/doc/rdiff-backup-$(COMPONENT_VERSION); \
+  done; \
+  $(CP) -p usr/lib/python$$V/vendor-packages/share/bash-completion/completions/rdiff-backup \
+    usr/share/bash-completion/completions; \
+  $(CP) -p usr/lib/python$$V/vendor-packages/share/man/man1/rdiff-backup-statistics.1 \
+    usr/share/man/man1; \
+  $(CP) -p usr/lib/python$$V/vendor-packages/share/man/man1/rdiff-backup.1 \
+    usr/share/man/man1; \
+done \
+)
```

Comment:
This works only for a single python version as I see it.
If we add 3.9 later the $(CP) commands will overwrite files from the former loop. Maybe we need a mediator?